api/v1beta1/remotesecret_types.go

```diff
@@ -158,10 +188,17 @@ type LinkableSecretSpec struct {
 	// Only kubernetes.io/service-account-token, kubernetes.io/dockercfg, kubernetes.io/dockerconfigjson and kubernetes.io/basic-auth
 	// are supported. All other secret types need to have their mapping specified manually using the Fields.
```

Comment:
Mentioning the "Fields" here is probably a remnant of the SPIAccessTokenBinding where this code originates from. Should we update the wording here, too? I don't think we put any constraints on the secret types in the remote secrets...